pkg/events/arm64.go

```diff
@@ -350,9 +350,6 @@ const (
 	MaxSyscallID             ID = 449
 )
 
-// TODO: There isnt a Unique32BitSyscalls list for arm64
```

Comment:
We should keep this comment to say that we have missing 32bit syscalls for arm64 compat mode

Reply:
"There are no 32bit only syscalls in arm64" and I put it under `MaxSyscallID` WDYT?

Reply:
But I don't think this statement is true.
I think there are 32bit syscalls for arm64, but we didn't implement it yet (this is why it was under "todo").